laws/src/main/scala/algebra/laws/RingLaws.scala

```diff
@@ -107,7 +107,15 @@ trait RingLaws[A] extends GroupLaws[A] {
     name = "ring",
     al = additiveCommutativeGroup,
     ml = multiplicativeMonoid,
-    parents = Seq(rig, rng)
+    parents = Seq(rig, rng),
+    "fromInt" -> forAll { (n: Int) =>
+      Ring.fromInt[A](n) ?== A.sumN(A.one, n)
+    },
+    "fromBigInt" -> forAll { (ns: List[Int]) =>
```

Comment:
why is this not
```scala
forAll { bi: BigInt =>
  Ring.fromBigInt[A](bi) ?== Ring.defaultFromBigInt[A](bi)
}
```
you have a kind of homomorphism here, that seems like a stronger law, isn't it?